worker/runc/runc.go

```diff
@@ -112,7 +112,7 @@ func NewWorkerOpt(root string, snFactory SnapshotterFactory, rootless bool, proc
 	opt = base.WorkerOpt{
 		ID:              id,
 		Labels:          xlabels,
-		MetadataStore:   md,
+		MdStore:         md,
```

Comment:
This rename shouldn't be needed, can just access `MetadataStore` property from embedded struct.

Reply:
Doh, yeah, that was silly again. Fixed https://github.com/moby/buildkit/pull/1627/commits/c04aeca088ef3d56d9289b21a37b24f1ff1c21d7